src/io/flutter/inspections/FlutterYamlNotificationProvider.java

```diff
@@ -64,6 +79,28 @@ public EditorNotificationPanel createNotificationPanel(@NotNull VirtualFile file
   }
 }
 
+class EnableFlutterSupportAction implements Runnable {
+  private final Module myModule;
+  private final Project myProject;
+
+  public EnableFlutterSupportAction(@NotNull final Module module, @NotNull Project project) {
+    myModule = module;
+    myProject = project;
+  }
+
+  @Override
+  public void run() {
+
+    myModule.setOption(Module.ELEMENT_TYPE, FlutterModuleType.getInstance().getId());
+
+    if (DartSdk.getDartSdk(myProject) != null && !DartSdkGlobalLibUtil.isDartSdkEnabled(myModule)) {
```

Comment:
How will this behave if there's no setting for a dart sdk? Perhaps only show the editor notification if there's not a flutter nature for the module, and we know that we can fix the issue (there's a flutter sdk defined)?